### DISCUSSION TITLE
Style in the text-editor

### DISCUSSION THREAD

**Discussion by AdrievandeVreede** [PrestaShop/PrestaShop#41071] [Ideas]
#### Is your feature request related to a problem?

It's always a disturbance to me that I can't see the style I see at the FO when I'm editing at the BO.
#### Describe the solution you'd like

Like in the FO, it has to be able to add a css-file to customise inside your text-editor. This way, it gives you the opportunity to look how your page will see at the FO. To be clear: **not the same as in the FO** 

#### Alternatives you've considered

First I thought that a link in the head of the editor could be a solution. But if you're having a template with complicated code then it can be a messy page. No option.

I hope to see it in the near future😃.

**Comment by florine2623** [comment]
Hello @AdrievandeVreede ,

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 

Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know!

Thank you

**Comment by hibatallahAouadni** [comment]
Hello @AdrievandeVreede 

Could you please edit your proposed solution cause it's kind of confused 🤔 do you mean that you need to customize your BO css like the FO or something else?
Please check and feedback.

Thanks!

**Comment by AdrievandeVreede** [comment]
Thank you for letting me know. I have changed the description of the solution. I hope it's more clear now!

**Comment by florine2623** [comment]
ping @AdrievandeVreede and @hibatallahAouadni 

From what I understood, it means being able to envision the changes directly in the BO.
For example, be able to see this :
<img width="1135" alt="Screenshot 2021-07-27 at 10 28 26" src="https://user-images.githubusercontent.com/16019289/127122391-8b4bcb71-11b5-49b7-bda0-52b241d76f2c.png">
Directly here :
<img width="778" alt="Screenshot 2021-07-27 at 10 28 20" src="https://user-images.githubusercontent.com/16019289/127122403-cfc50015-1f58-47cf-a2e9-9e6b60214306.png">
 
Is that right @AdrievandeVreede ?

**Comment by AdrievandeVreede** [comment]
That's right
